redis/src/cluster.rs

```diff
@@ -238,7 +238,7 @@ where
             connections: RefCell::new(HashMap::new()),
             slots: RefCell::new(SlotMap::new(cluster_params.read_from_replicas)),
             auto_reconnect: RefCell::new(true),
-            read_timeout: RefCell::new(None),
+            read_timeout: RefCell::new(cluster_params.response_timeout),
```

Comment:
Do we actually need `read_timeout`? Should it just be removed in favor of `cluster_params.response_timeout`?

Reply:
I guess we can remove it - I tried to avoid a larger change.

Reply:
Ok, avoiding the larger change is probably fine then, or maybe worth considering in the separate PR.